snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/SnomedPlugin.java

```diff
@@ -231,7 +233,12 @@ protected ComponentRevisionConflictProcessor getComponentRevisionConflictProcess
 	
 	@Override
 	protected SetMemberSearchRequestEvaluator getMemberSearchRequestEvaluator() {
-		return new SnomedRefSetMemberSearchRequestEvaluator();
+		return new SnomedMemberSearchRequestEvaluator();
+	}
+	
+	@Override
+	protected SetMappingSearchRequestEvaluator getMappingSearchRequestEvaluator() {
+		return new SnomedRefSetMappingSearchRequestEvaluator();
```

Comment:
Rename this to `SnomedConceptMapSearchRequestEvaluator`.

Reply:
Done